src/components/common/BottomNavigation/button.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { HTMLAttributes, ReactNode } from 'react';
+
+interface BottomNaviagtionButtonProps
+  extends HTMLAttributes<HTMLButtonElement> {
+  children: ReactNode;
+}
+
+export default function BottomNaviagtionButton({
+  children,
+  onClick,
+}: BottomNaviagtionButtonProps) {
+  return (
+    <button
+      className="flex justify-center items-center w-[56px] h-[56px] rounded-[50%] bg-bottom-navigation-button border-gray-900 border-[1px] shadow-bottom-navigation-button"
+      onClick={onClick}
+      type="button"
+    >
+      {children}
+    </button>
```

Comment:
버튼에는 이미 children prop이 포함되어 있습니다!

```suggestion
export default function BottomNaviagtionButton({
className, children, ...restProps
}: PropsWithChildren<HTMLButtonElement>) {
  return (
    <button
      className={cn("flex justify-center items-center w-[56px] h-[56px] rounded-[50%] bg-bottom-navigation-button border-gray-900 border-[1px] shadow-bottom-navigation-button", className)}
      type="button"
{...restProps}
    >
      {children}
    </button>
```